.github/workflows/update_cross_build_releases.yml

```diff
@@ -6,9 +6,12 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: "Version number"
+        description: "Version number to update to. This should not contain the 'v' prefix, since our releases on GitHub do not have it."
         required: true
 
+env:
+  version: ${{ github.event_name == 'release' && github.event.release.name || github.event.inputs.version }}
```

Comment:
I used `name` before, but it didn't work, so I replaced it with `tag_name` in https://github.com/pyodide/pyodide/pull/4820. 0.26.3 patch release does not include this commit so it probably used `name` and failed.

I am not exactly sure about the difference between the `name` and `tag_name` though. But could you please double check it?

Reply:
I'll double-check it – sure! I'm convinced that I used `release.name` yesterday and created a tag + a release from it on my fork, which worked (as seen from the PRs that were opened in my fork in the PR description). But since #4820 wasn't included in 0.26.3 and I can't find the code for what @pyodide-ci-bot is doing, let's go with `tag_name` as you suggest, since that will be a safer way.

Also, looking at https://github.com/pyodide/pyodide/actions/workflows/update_cross_build_releases.yml, it failed for all 0.26.X releases and all of the runs used `release.name`, it makes sense to switch.

Reply:
```suggestion
  version: ${{ github.event_name == 'release' && github.event.release.tag_name || github.event.inputs.version }}
```

Reply:
Addressed in 3808d351668903d22fda116de69ee80574e58c1f. Thanks!